docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.9"
```

Comment:
please remove this, we know this is not going to work as, none of our dependencies and configs are present.

Reply:
Yup, will do, sorry by mistake this got added, I thought of removing this before the scrum call.